recipes/weasyprint/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "WeasyPrint" %}
+{% set version = "v47" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Kozea/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 707b3b54c6552c99f67682178316b653ed229b03ffb17158f7374e99829338eb
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python>3.5
+    - pip
+    - setuptools>=39.2.0
+    - cffi>=0.6
+    - html5lib>=0.999999999
+    - cairocffi>=0.9.0
+    - tinycss2>=1.0.0
+    - cssselect2>=0.1
+    - CairoSVG>=2.4.0
+    - Pyphen>=0.8
+    - pytest-runner
+    
+  run:
+    - python>3.5
+    - setuptools>=39.2.0
+    - cffi>=0.6
+    - html5lib>=0.999999999
+    - cairocffi>=0.9.0
+    - tinycss2>=1.0.0
+    - cssselect2>=0.1
+    - CairoSVG>=2.4.0
+    - Pyphen>=0.8
+
+test:
+  imports:
+    - weasyprint
+
+about:
+  home: https://github.com/Kozea/WeasyPrint
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'WeasyPrint converts web documents (HTML with CSS, SVG, …) to PDF'
+
+  description: |
+    WeasyPrint is a smart solution helping web developers to create PDF
+    documents. It turns simple HTML pages into gorgeous statistical
+    reports, invoices, tickets…
+
+  doc_url: https://weasyprint.readthedocs.io/en/stable/
+  dev_url: https://github.com/Kozea/WeasyPrint
+
+extra:
+  recipe-maintainers:
+    - mrandrewandrade
+    - liZe
```

Comment:
@liZe are you ok with being a maintainer?

Reply:
He agreed here: https://github.com/conda-forge/tinycss2-feedstock/pull/3#issuecomment-497854574

Reply:
@liZe agreed here: https://github.com/conda-forge/tinycss2-feedstock/pull/3#issuecomment-497854574